docs/api/role-management.asciidoc

```diff
@@ -0,0 +1,23 @@
+[role="xpack"]
+[[role-management-api]]
+== Role Management API
+
+[NOTE]
+.Stable API
```

Comment:
Potentially? Since we're using these APIs to manage non-beta features, it'd be nice not to have to. Which changes were you looking to make for Spaces? I thought we were going to be "adding" to the privilege entry, so instead of doing:
```
{
  "kibana": [{ "privileges": ["read"] ]
}
```

we'd do:
```
{
  "kibana": [{ "privileges": ["read"], spaces: ["marketing" ] ]
}
```

Reply:
Yup, the lack of a spaces entry would mean "*", so they'd only be additive changes. I was trying to mirror the structure of the Elasticsearch Role `indices` section with the way it's structured, and it felt closer to the way the application entries themselves are represented on the Elasticsearch role (because they aren't grouped by resource and technically you can assign multiple resources to a single application entry).